src/cli.js

```diff
@@ -7,6 +7,12 @@
 
 // babel-preset-env will transform the line below into
 // individual requires for babel-polyfill based on environment
-require('babel-polyfill');
+require('babel-polyfill')
 
-require('./cliEntry')(process.argv.slice(2));
+// dump start parameters
+const isVerbose = process.argv.includes('--verbose')
```

Comment:
I think we have better ways to parse CLI args?

Reply:
Any suggestions? I know that it's quite a dirty approach.